monkey/tests/unit_tests/infection_monkey/ransomware/test_ransomware_payload.py

```diff
@@ -66,66 +37,71 @@ def ransomware_payload(ransomware_payload_config):
     return RansomewarePayload(ransomware_payload_config)
 
 
-def test_file_with_excluded_extension_not_encrypted(tmp_path, ransomware_payload):
+def test_file_with_excluded_extension_not_encrypted(ransomware_target, ransomware_payload):
     ransomware_payload.run_payload()
 
-    assert hash_file(tmp_path / TEST_LIB_DLL) == TEST_LIB_DLL_CLEARTEXT_SHA256
+    assert hash_file(ransomware_target / TEST_LIB_DLL) == TEST_LIB_DLL_CLEARTEXT_SHA256
 
 
-def test_shortcut_not_encrypted(tmp_path, ransomware_payload):
+def test_shortcut_not_encrypted(ransomware_target, ransomware_payload):
     ransomware_payload.run_payload()
 
-    assert hash_file(tmp_path / SHORTCUT_LNK) == SHORTCUT_LNK_CLEARTEXT_SHA256
+    assert hash_file(ransomware_target / SHORTCUT_LNK) == SHORTCUT_LNK_CLEARTEXT_SHA256
 
 
 @pytest.mark.skipif(
     os.name == "nt" and not is_user_admin(), reason="Test requires admin rights on Windows"
 )
-def test_symlink_not_encrypted(tmp_path, ransomware_payload):
+def test_symlink_not_encrypted(ransomware_target, ransomware_payload):
     SYMLINK = "symlink.pdf"
-    link_path = tmp_path / SYMLINK
-    link_path.symlink_to(tmp_path / TEST_LIB_DLL)
+    link_path = ransomware_target / SYMLINK
+    link_path.symlink_to(ransomware_target / TEST_LIB_DLL)
 
     ransomware_payload.run_payload()
 
-    assert hash_file(tmp_path / SYMLINK) == TEST_LIB_DLL_CLEARTEXT_SHA256
+    assert hash_file(ransomware_target / SYMLINK) == TEST_LIB_DLL_CLEARTEXT_SHA256
 
 
-def test_encryption_not_recursive(tmp_path, ransomware_payload):
+def test_encryption_not_recursive(ransomware_target, ransomware_payload):
     ransomware_payload.run_payload()
 
-    assert hash_file(tmp_path / SUBDIR / HELLO_TXT) == HELLO_TXT_CLEARTEXT_SHA256
+    assert hash_file(ransomware_target / SUBDIR / HELLO_TXT) == HELLO_TXT_CLEARTEXT_SHA256
 
 
-def test_file_with_included_extension_encrypted(tmp_path, ransomware_payload):
-    assert hash_file(tmp_path / ALL_ZEROS_PDF) == ALL_ZEROS_PDF_CLEARTEXT_SHA256
-    assert hash_file(tmp_path / TEST_KEYBOARD_TXT) == TEST_KEYBOARD_TXT_CLEARTEXT_SHA256
+def test_file_with_included_extension_encrypted(ransomware_target, ransomware_payload):
+    assert hash_file(ransomware_target / ALL_ZEROS_PDF) == ALL_ZEROS_PDF_CLEARTEXT_SHA256
+    assert hash_file(ransomware_target / TEST_KEYBOARD_TXT) == TEST_KEYBOARD_TXT_CLEARTEXT_SHA256
 
     ransomware_payload.run_payload()
 
-    assert hash_file(tmp_path / with_extension(ALL_ZEROS_PDF)) == ALL_ZEROS_PDF_ENCRYPTED_SHA256
     assert (
-        hash_file(tmp_path / with_extension(TEST_KEYBOARD_TXT))
+        hash_file(ransomware_target / with_extension(ALL_ZEROS_PDF))
+        == ALL_ZEROS_PDF_ENCRYPTED_SHA256
+    )
+    assert (
+        hash_file(ransomware_target / with_extension(TEST_KEYBOARD_TXT))
```

Comment:
Why are we testing this with two files?

Reply:
Same question for `test_file_encrypted_in_place()`.

Reply:
I want to test that all (more than 1) expected files are encrypted. I could change the test name to clarify, or split this up into two tests.